client/components/ChartBox.tsx

```diff
@@ -287,19 +288,23 @@ const ChartBox: React.FC<IProps> = ({ queries }) => {
             chartSix: 'block',
         });
     };
+    const multipleFnOne = (num: any) => {
+        setDaysFn(num);
+        setTimeRangeDays(num);
+    };
     return (
         <div id="chartBoxInside">
             <div className="projectSelector">
-                <button onClick={() => setTimeRangeDays(1)} className="chartBtn" type="button">
+                <button onClick={() => multipleFnOne(1)} className="chartBtn" type="button">
                     Last 24 h
                 </button>
-                <button onClick={() => setTimeRangeDays(7)} className="chartBtn" type="button">
+                <button onClick={() => multipleFnOne(7)} className="chartBtn" type="button">
                     Last Week
                 </button>
-                <button onClick={() => setTimeRangeDays(30)} className="chartBtn" type="button">
+                <button onClick={() => multipleFnOne(30)} className="chartBtn" type="button">
                     Last Month
                 </button>
-                <button onClick={() => setTimeRangeDays(360)} className="chartBtn" type="button">
+                <button onClick={() => multipleFnOne(360)} className="chartBtn" type="button">
```

Comment:
quick question, why do we have 360 here instead of 365?

Reply:
Good question, I initially set them up to be increments of 30 (30, 60, 90, 180). We changed the scale but I still kept them as increments of 30. It does probably make more sense for this to be 365.

Reply:
I have changed it to 365. It was already set to 365 in a different spot so worth the review.